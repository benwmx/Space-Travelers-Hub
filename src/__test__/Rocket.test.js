import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../redux/configureStore';
import Rocket from '../components/Rockets';

describe('Rocket Component test', () => {
  it('Should render', () => {
    const app = renderer.create(
      <Provider store={store}>
        <Rocket />
      </Provider>,
    ).toJSON();
    expect(app).toMatchSnapshot();
  });
});
