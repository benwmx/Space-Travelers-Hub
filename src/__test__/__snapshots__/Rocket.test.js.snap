// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rocket Component test Should render 1`] = `
<div
  className="rocketsContainer"
>
  <div
    className="rocket"
  >
    <div>
      <img
        alt="Rocket"
        className="image"
        src=""
      />
    </div>
    <div
      className="info"
    >
      <h2>
        
      </h2>
      <p
        className="description"
      >
        <span />
        
      </p>
      <button
        className="reserveRocket"
        id=""
        onClick={[Function]}
        type="button"
      >
        Reserve Rocket
      </button>
    </div>
  </div>
</div>
`;
